rootx/src/rootx.cxx

```diff
@@ -201,6 +201,9 @@ static void SetLibraryPath()
 #  endif
    putenv(msg);
 #ifdef ROOTPREFIX
+   } else {
+      char *msg = strdup("LD_LIBRARY_PATH=" ROOTLIBDIR);
```

Comment:
Doesn't this overwrite the `LD_LIBRARY_PATH` for the current process? If this is really needed after the other commits, we should probably try to append or prepend to `LD_LIBRARY_PATH` and think about if Mac/Windows also need to do something here.

Reply:
That's fixed in https://github.com/root-project/root/pull/5145 - thanks for the comment!